.travis.yml

```diff
@@ -11,13 +11,13 @@ script: echo "Running tests against $(node -v)..." && yarn test
 jobs:
   include:
     - stage: Build
-      node_js: node
+      node_js: 10
```

Comment:
😢 Necessary since we're so behind on libraries

Reply:
@rkuykendall why? what changed in this PR that makes this necessary?

Reply:
Nothing in this PR. The change was node steadily advancing while we're stuck in the past. So I had to lock our build step to node 10 to fix builds again, until we do the update.